src/main/java/mate/academy/hibernate/relations/model/Movie.java

```diff
@@ -2,43 +2,33 @@
 
 import java.util.ArrayList;
 import java.util.List;
-
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToMany;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
 public class Movie implements Cloneable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "movie_seq")
     private Long id;
     private String title;
+    @ManyToMany
```

Comment:
let's for now fix LazyInitializationException by adding `fetch = FetchType.EAGER`
we will later learn a better way to fix it